staging/src/k8s.io/apimachinery/pkg/apis/meta/fuzzer/fuzzer.go

```diff
@@ -248,33 +249,32 @@ func v1FuzzerFuncs(codecs runtimeserializer.CodecFactory) []interface{} {
 				metav1.LabelSelectorOpDoesNotExist,
 			}
 
-			if j.MatchExpressions != nil {
```

Comment:
I know the nil check is not needed, but would you mind restoring it to minimize the diff? I'd like to see this picked back to release branches to deflake there as well

Reply:
lgtm otherwise

Reply:
Couldn't resist. Sure, done.